corehq/apps/users/models.py

```diff
@@ -2433,6 +2433,12 @@ def get_dimagi_emails_by_domain(cls, domain):
             if user_doc['email'].endswith('@dimagi.com'):
                 yield user_doc['email']
 
+    def save(self, fire_signals=True, **params):
+        super().save(fire_signals=fire_signals, **params)
+        from corehq.apps.callcenter.tasks import sync_web_user_usercases_if_applicable
+        for domain in self.get_domains():
+            sync_web_user_usercases_if_applicable(self, domain, spawn_task=True)
```

Comment:
`CommCareUser.save` only calls `sync_usercases_if_applicable` if `fire_signals` is true. I'd expect this to behave the same.